### PR TITLE
cleaned project tree and test forking

### PR DESCRIPTION
**Pull request by fakhbari-dev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 </head>
 <body>
     <h1>** Empty Start Project **</h1>
+    <p>start Now!</p>
```
Comment (fabidev1993) [inline index.html]: it is not requierd. please remove it

Comment (fakhbari-dev) [reply index.html]: ok i got it

**Comment by fakhbari-dev** [comment]
i did soo please check

**Comment by fabidev1993** [comment]
merged first fork test

**Pull request merged**